tests/containers/validate_btrfs.pm

```diff
@@ -55,12 +63,16 @@ sub _test_btrfs_device_mgmt {
     my $btrfs_head = '/tmp/subvolumes_saved';
     record_info "test btrfs";
     script_run("df -h");
-    # /var is using its own partition which is size of 10G. Create file in the container
-    # enough to fill up the partition up to ~99%
-    $rt->up('huge_image', keep_container => 1, cmd => 'fallocate -l 9149000KiB bigfile.txt');
+    # Determine the remaining size of /var
+    script_run("df");
```

Comment:
is this for debug purposes only? Maybe we could dump the output with `record_info` or something

Reply:
Ah damn, this was forgotten :sweat_smile: Cleaned it up, thanks for the note!